src/components/WarningAnimateMessage/index.js

```diff
@@ -55,43 +55,41 @@ const WarningAnimatedMessage = ({isSHow, top = 100, left = 110}) => {
   }, [isRunAnimate]);
 
   React.useEffect(() => {
-    getSpecificCache(OPEN_FIRST_TIME, (cache) => {
-      if (!cache) setIsRunAnimated(isSHow);
-    });
-  }, [isSHow]);
+    if (StorageUtils.incognitoCreatePostFirstTime.get() !== 'true') {
+      setIsRunAnimated(isShow);
+    }
+  }, [isShow]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useEffect` hook has been updated to use `StorageUtils.incognitoCreatePostFirstTime.get() !== 'true'` instead of `getSpecificCache(OPEN_FIRST_TIME, (cache) => { if (!cache) setIsRunAnimated(isSHow); });`. This change simplifies the logic and improves readability by directly checking the value from storage.
```diff
- getSpecificCache(OPEN_FIRST_TIME, (cache) => {
-   if (!cache) setIsRunAnimated(isSHow);
- });
+ if (StorageUtils.incognitoCreatePostFirstTime.get() !== 'true') {
+   setIsRunAnimated(isShow);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->